src/main/java/frc/robot/Robot.java

```diff
@@ -35,6 +43,12 @@ public void robotInit() {
         // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
         // autonomous chooser on the dashboard.
         m_robotContainer = new RobotContainer();
+
+        mCompressor = new Compressor(0);
+        mCompressor.setClosedLoopControl(true);
+
+        mSolenoidOne.set(DoubleSolenoid.Value.kForward);
+        mSolenoidTwo.set(DoubleSolenoid.Value.kForward);
```

Comment:
You want to set the pneumatics off or else they might fire the moment we connect to the robot (which is very unsafe).